src/i18n.ts

```diff
@@ -15,7 +15,7 @@ i18n.use(initReactI18next).init({
     },
   },
   lng: 'ru',
-  fallbackLng: 'en',
+  fallbackLng: ['en', 'ru'],
```

Comment:
Are u sure that need to add all languages to fallback?

Reply:
I`ll change it to a more appropriate variant in future pr if I find it. For now it works